browser/services/platform.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
 
 class Platform {
+
+  data(installerDataSvc){
```

Comment:
I think you can safely make this static so you don't need to instantiate the class every time just for the sake of calling the method.